app/src/main/res/layout/grid_item_addon.xml

```diff
@@ -21,34 +21,49 @@
     android:layout_height="wrap_content"
     style="@style/Widget.CardView">
 
-    <LinearLayout
+    <RelativeLayout
+        xmlns:android="http://schemas.android.com/apk/res/android"
         android:layout_width="match_parent"
-        android:layout_height="wrap_content"
-        android:orientation="horizontal">
+        android:layout_height="wrap_content">
 
         <ImageView
             android:id="@+id/art"
             android:layout_width="@dimen/addonlist_art_width"
             android:layout_height="@dimen/addonlist_art_heigth"
-            android:contentDescription="@string/poster"/>
+            android:layout_alignParentStart="true"
+            android:layout_alignParentLeft="true"
+            android:contentDescription="@string/poster"
+            android:scaleType="centerCrop"/>
 
-        <LinearLayout
-            android:layout_width="match_parent"
-            android:layout_height="match_parent"
-            android:orientation="vertical"
-            android:gravity="center_vertical">
             <TextView
                 android:id="@+id/title"
                 android:layout_width="match_parent"
                 android:layout_height="wrap_content"
-                style="@style/TextAppearance.Medialist.Title"
-                android:paddingTop="0dp"/>
+                android:layout_toRightOf="@id/art"
+                android:layout_toEndOf="@id/art"
+                android:layout_alignTop="@id/art"
+                style="@style/TextAppearance.Medialist.Title"/>
+
             <TextView
                 android:id="@+id/details"
                 android:layout_width="match_parent"
                 android:layout_height="wrap_content"
+                android:layout_alignLeft="@id/title"
+                android:layout_alignStart="@id/title"
+                android:layout_below="@id/title"
                 style="@style/TextAppearance.Medialist.Details"
                 android:paddingBottom="0dp"/>
-        </LinearLayout>
-    </LinearLayout>
+
+            <ImageView
```

Comment:
Instead of showing a cancel icon, why not grey-out the whole item? That would make it more consistent with how Android in general displays disabled UI elements.

Reply:
I'll see what I can do :)

Reply:
I'm not sure this is the best idea, the subtext is already grey (night theme) and you can still click and go to the addon details screen.
Making it look disabled would cause the users to think it cannot be clicked.

Reply:
Ah yes, I see. Why not show the text "disabled" instead?
I think the icon is confusing. I feel the same about the checkmark icon shown on the Addon detail page. @SyncedSynapse do you have any ideas?

Reply:
Not really. I can't think of a proper icon for enable/disable but i actually think that the cross icon on this PR is better than the checkmark i used on the details page.